rtengine/dcraw.h

```diff
@@ -76,6 +76,20 @@ class DCraw
     }
 
 protected:
+    enum CropMode : std::uint_fast16_t {                                         // RT
```

Comment:
Don't import short symbols like `NA` into the surrounding namespace. Better use `enum class`.